Plugin/src/SofaPython3/DataHelper.cpp

```diff
@@ -45,25 +45,36 @@ std::string toSofaParsableString(const py::handle& p)
     if(py::isinstance<py::list>(p) || py::isinstance<py::tuple>(p))
     {
         std::stringstream tmp;
-        for(auto pa : p){
+        for(auto pa : p)
+        {
             tmp << toSofaParsableString(pa) << " ";
         }
         return tmp.str();
     }
     //TODO(dmarchal) This conversion to string is so bad.
     if(py::isinstance<py::str>(p))
+    {
         return py::str(p);
+    }
+
     return py::repr(p);
 }
 
 /// RVO optimized function. Don't care about copy on the return code.
-void fillBaseObjectdescription(sofa::core::objectmodel::BaseObjectDescription& desc,
+py::list fillBaseObjectdescription(sofa::core::objectmodel::BaseObjectDescription& desc,
```

Comment:
To keep consistency I recommand outputing  the two lists (desc and dataParents) with non const methods parameters and keep the function "void".